TrafficCapture/trafficCaptureProxyServer/src/main/resources/log4j2.properties

```diff
@@ -6,7 +6,8 @@ appender.console.type = Console
 appender.console.name = STDERR
 appender.console.target = SYSTEM_ERR
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} [%t] %c{1} - %msg%equals{ ctx=%mdc}{ ctx=\{\}}{}%n
+appender.console.layout.pattern = [%-5level] %d{DEFAULT_MICROS}{UTC} [%t] %c{1} - %msg%equals{ ctx=%mdc}{ ctx=\{\}}{}%n
+appender.console.direct = true
```

Comment:
Not sure this topology is fully aligned, but the way the `serviceContainer` is setup is below to blocking which would cause tiny write to be committed immediately and could show up in strange slow down.

Reply:
Existing Data Flow  (All Blocking Calls):
     Application → Log4j2 → Standard Out → ECSLogDriver → Cloudwatch Logs


New Data Flow (Blocking unless specified):
     Application → [AsyncLogger](https://github.com/apache/logging-log4j2/blob/f0bd87818d43efa8337f56526ced2020c308d9bf/log4j-core/src/main/java/org/apache/logging/log4j/core/async/AsyncLogger.java#L66) (async)→ Log4j2 → ECSLogDriver → Cloudwatch Logs

The AsyncLogger internally buffers so the rest can be "blocking" and still not block the application flow